client/public/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <title>Engage Boilerplate</title>
+    <base href="/" />
```

Comment:
this is not needed, and its opinionated enough to actually break the app's relative file paths more often than not. remove.